src/components/infoTooltips/BorrowCapTooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import { ExclamationIcon } from '@heroicons/react/outline';
+import { Trans } from '@lingui/macro';
+import { Box } from '@mui/material';
+import { AssetCapData } from 'src/hooks/getAssetCapUsage';
+import { TextWithTooltip, TextWithTooltipProps } from '../TextWithTooltip';
+import { Link } from '../primitives/Link';
+
+type BorrowCapTooltipProps = TextWithTooltipProps & {
+  borrowCap: AssetCapData;
+};
+
+export const BorrowCapTooltip = ({ borrowCap, ...rest }: BorrowCapTooltipProps) => {
+  // Don't show a tooltip when less than 98% utilized
+  if (borrowCap.percentUsed < 98) return null;
```

Comment:
To switch this from a fixed 98% display cap to a user-based display, then it probably makes sense to check this in `getAssetCapUsage` and pass an enum prop to the tooltip either:

'none': No cap on the asset
'negligent': There is a cap but will not affect user
'partial': Remaining cap or ceiling will affect user max supply or borrow
'blocked': Remaining cap or ceiling is 0 or asset frozen

Reply:
Then it can be re-used for debt ceiling and frozen as well